Simplenote/Classes/SPMarkdownParser.m

```diff
@@ -29,22 +29,40 @@ + (NSString *)renderHTMLFromMarkdownString:(NSString *)markdown
     hoedown_html_renderer_free(renderer);
     
     NSString *htmlString = [[NSString alloc] initWithData:htmlData encoding:NSUTF8StringEncoding];
+    
+    NSRegularExpression *regex = [NSRegularExpression regularExpressionWithPattern:
+                                  @"\n\\s+- \\[(x|X|o|o| )\\]\\s+([^\n]+)" options:0 error:nil];
```

Comment:
Would it be possible to add a brief comment, documenting what does this one do?

Thanks!!!